frontend/server/libs/dao/Users_Badges.dao.php

```diff
@@ -18,4 +12,33 @@
   *
   */
 class UsersBadgesDAO extends UsersBadgesDAOBase {
+    private static function getUserOwnedBadges(str $userId) {
```

Comment:
```suggestion
    private static function getUserOwnedBadges(Users $user) {
```

puedes obtenerlo usando `$r['current_user']`.